site/content/en/reference/pkg/fix/_index.md

```diff
@@ -0,0 +1,41 @@
+---
+title: "Fix"
+linkTitle: "fix"
+type: docs
+description: >
+   Fix a local package.
+---
+<!--mdtogo:Short
+    Fix a local package.
```

Comment:
Can we expand a little bit here to make it clear what we mean by "fix"

Reply:
Done. Followed the short pattern like other commands, but expanded it  now.